doc/classes/ProjectSettings.xml

```diff
@@ -1008,8 +1008,10 @@
 		<member name="display/window/size/initial_position_type" type="int" setter="" getter="" default="1">
 			Main window initial position.
 			[code]0[/code] - "Absolute", [member display/window/size/initial_position] is used to set window position.
-			[code]1[/code] - "Primary Screen Center".
-			[code]2[/code] - "Other Screen Center", [member display/window/size/initial_screen] is used to set the screen.
+			[code]1[/code] or [code]2[/code] - "Primary Screen Center".
```

Comment:
`2` is the `center of the main window`, and is an invalid value for the initial main window position. It is handled the same way as `1`, so technically correct, but probably should not be listed in the documentation.


```suggestion
			[code]1[/code] - "Primary Screen Center".
```